src/styles/blocks/stopwatch.scss

```diff
@@ -0,0 +1,68 @@
+.stopwatch {
+  width: 400px;
+  height: 400px;
+  border: 15px solid $main-color;
+  border-radius: 100%;
+  background-color: $background-color;
+
+  position: relative;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  margin: 40px auto;
+
+  &__center {
+    width: 30px;
+    height: 30px;
+    background-color: $main-color;
+    border-radius: 100%;
+    z-index: 3;
+  }
+
+  &__sec {
+    width: 10px;
+    height: 200px;
+    background-color: $sec-hand-color;
+    top: 0;
+    position: absolute;
+    z-index: 1;
+    animation-name: rotation-sec;
+    animation-duration: 5s;
+    animation-timing-function: linear;
+  }
+
+  &__min {
+    width: 15px;
+    height: 100px;
+    background-color: $min-hand-color;
+    top: 0;
+    position: absolute;
+    z-index: 2;
+    transform: translateY(100px);
+    animation-name: rotation-min;
+    animation-duration: 60s;
+    animation-timing-function: linear;
+  }
```

Comment:
arrows has almost same styles. suggest to make mixin for them which will accept some arguments of styles which different.